Modules/_testcapimodule.c

```diff
@@ -2587,6 +2587,117 @@ get_datetime_fromtimestamp(PyObject* self, PyObject *args)
     return rv;
 }
 
+static PyObject *
+get_fields_from_date(PyObject* self, PyObject *args)
```

Comment:
```suggestion
get_fields_from_date(PyObject *self, PyObject *args)
```

Same change applies to all the other C functions - the `*` needs to be grouped with the variable name.